packages/perseus/src/types.ts

```diff
@@ -589,6 +595,12 @@ export type WidgetExports<
      */
     scorer?: WidgetScorerFunction;
 
+    /**
+     * A function that provides a public version of the widget options that can
+     * be shared with the client.
+     */
+    getPublicOptions?: PublicWidgetOptionsFunctions;
```

Comment:
Would this be more clear if it was `getPublicWidgetOptions`? I think I was trying to make it a bit shorter, but after looking at it again not sure if it's clear enough.

Reply:
This was updated to `splitWidgetOptions`.